packages/manager/modules/iam/src/components/actionSelect/actionSelect.controller.js

```diff
@@ -442,6 +447,17 @@ export default class ActionSelectController {
         (currentActionTree) => currentActionTree.value === actionTree.value,
       )
     ] = shadowActionTree;
+
+    // Check if any action of this category contains the searchQuery string
+    this.showSearchNotFoundError = !actionTree.categories
+      .reduce(
+        (result, category) => [
+          ...result,
+          ...category.actions.map(({ value }) => value),
+        ],
+        [],
+      )
+      .some((action) => action.indexOf(actionTree.searchQuery) > -1);
```

Comment:
Why do you call reduce here ?
It would be more efficient if you directly call some on actionTree.categories

Reply:
Done